policies/compliance-docs/dr-exercise-recent.py

```
@@ -0,0 +1,42 @@
+
+def main(node=None, max_days_override=None):
+    c = Check("dr-exercise-recent", "DR exercise should be conducted regularly", node=node)
+    with c:
+        max_days_str = max_days_override if max_days_override is not None \
+            else variable_or_default("max_days_since_exercise", "365")
+        try:
+            max_days = int(max_days_str)
+        except (ValueError, TypeError):
+            raise ValueError(
+                f"Policy misconfiguration: 'max_days_since_exercise' must be a number, got '{max_days_str}'"
+            )
```

Ruff (0.15.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)